external-crates/move/crates/move-analyzer/tests/cursor_tests/cursor_dot_call_tests.exp

```diff
@@ -148,13 +148,13 @@ cursor info:
 - module: Move2024::M2
 - definition: function baz
 - position: exp
-- value: DotCall(Name(Single(PathEntry { name: "some_struct", tyargs: None, is_macro: None })), "f3", None, None, [Name(Single(PathEntry { name: "val", tyargs: None, is_macro: None }))])
```

Comment:
These changes are unrelated and already landed in https://github.com/MystenLabs/sui/pull/20547

Once it got added to the PR, though, I wasn't sure how to cleanly remove this file....